[2024-09-29T03:14:18.529+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-29T03:14:18.558+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_cron_expression_v03.task1 manual__2024-09-29T03:14:17.856663+00:00 [queued]>
[2024-09-29T03:14:18.573+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_cron_expression_v03.task1 manual__2024-09-29T03:14:17.856663+00:00 [queued]>
[2024-09-29T03:14:18.573+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-09-29T03:14:18.588+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): task1> on 2024-09-29 03:14:17.856663+00:00
[2024-09-29T03:14:18.613+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=17838) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-29T03:14:18.603+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_cron_expression_v03', 'task1', 'manual__2024-09-29T03:14:17.856663+00:00', '--job-id', '273', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_cron_expression.py', '--cfg-path', '/tmp/tmpazmn50ap']
[2024-09-29T03:14:18.616+0000] {standard_task_runner.py:72} INFO - Started process 17839 to run task
[2024-09-29T03:14:18.617+0000] {standard_task_runner.py:105} INFO - Job 273: Subtask task1
[2024-09-29T03:14:18.670+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_cron_expression_v03.task1 manual__2024-09-29T03:14:17.856663+00:00 [running]> on host f9a59cccd6d1
[2024-09-29T03:14:18.721+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='dag_with_cron_expression_v03' AIRFLOW_CTX_TASK_ID='task1' AIRFLOW_CTX_EXECUTION_DATE='2024-09-29T03:14:17.856663+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-29T03:14:17.856663+00:00'
[2024-09-29T03:14:18.722+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-29T03:14:18.736+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-29T03:14:18.737+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo dag with cron expression']
[2024-09-29T03:14:18.752+0000] {subprocess.py:86} INFO - Output:
[2024-09-29T03:14:18.760+0000] {subprocess.py:93} INFO - dag with cron expression
[2024-09-29T03:14:18.760+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-29T03:14:18.779+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-29T03:14:18.779+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_cron_expression_v03, task_id=task1, run_id=manual__2024-09-29T03:14:17.856663+00:00, execution_date=20240929T031417, start_date=20240929T031418, end_date=20240929T031418
[2024-09-29T03:14:18.803+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-29T03:14:18.817+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-29T03:14:18.818+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
